go.mod

```diff
@@ -51,7 +50,6 @@ require (
 	github.com/Azure/go-amqp v1.0.0 // indirect
 	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/btcsuite/btcd v0.20.1-beta // indirect
```

Comment:
good riddance to this - it was singlehandedly causing multiple dpendabot alerst in all repositories